mlrun/datastore/targets.py

```diff
@@ -1413,6 +1413,27 @@ def as_df(self, columns=None, df_module=None, **kwargs):
 
 
 class KafkaTarget(BaseStoreTarget):
+    """Kafka target storage driver, used to materialize feature set
+     data into kafka topics.
+
+    example::
+
+        # define target
+        kafka_target = KafkaTarget(
+            "kafka",
+            path='my_topic',
+            bootstrap_servers='localhost:9092'
+        )
+
+        # ingest
+        stocks_set.ingest(stocks, [kafka_target])
```

Comment:
Maybe it's better not to mention feature store here, since datastore can be used in other contexts as well, e.g. in serving functions. This applies to the docstring topic line as well.

Reply:
I removed that from docstring, but I see that example show only one possible use case from a lot. Is it fine?